### PR TITLE
Bundling dependabot PRs

### PR DESCRIPTION
**Pull request by Skarlso**
Bundling together prs: 
https://github.com/weaveworks/eksctl/pull/5171
https://github.com/weaveworks/eksctl/pull/5170

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,20 +8,20 @@ go 1.18
 require (
 	github.com/Masterminds/semver/v3 v3.1.1
 	github.com/aws/amazon-ec2-instance-selector/v2 v2.0.4-0.20220124212200-2aee60ac608e
-	github.com/aws/aws-sdk-go v1.43.45
-	github.com/aws/aws-sdk-go-v2 v1.16.2
+	github.com/aws/aws-sdk-go v1.43.43
```
Comment (cPu1) [inline go.mod:11]: This is downgraded from v1.43.45.

**Review by cPu1 — COMMENTED** [review]
There are a few modules that are downgraded in this PR. Is this changelist outdated?

**Comment by Skarlso** [comment]
Yeah, I just noticed. Something didn't quite work out. :) I'll investigate after work. :) thanks Chetan!

**Comment by Skarlso** [comment]
Yeah, I just noticed. Something didn't quite work out. :) I'll investigate after work. :) thanks Chetan!

**Pull request closed without merging**